doc/frameworks/tensorflow/using_tf.rst

```diff
@@ -197,6 +197,27 @@ The following args are not permitted when using Script Mode:
 Where the S3 url is a path to your training data within Amazon S3.
 The constructor keyword arguments define how SageMaker runs your training script.
 
+Specify a Docker image using an Estimator
+-----------------------------------------
+
+You can also specify a Docker image when creating an Estimator by specifying the ECR URI for the Python and framework version directly. For a full list of available container URIs, see `Available Deep Learning Containers Images <https://github.com/aws/deep-learning-containers/blob/master/available_images.md>`__ For more information on using Docker containers, see `Use Your Own Algorithms or Models with Amazon SageMaker <https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms.html>`__.
+
+When specifying the image, you must use the ``image_name=''`` arg to replace both of the following args:
+
+- ``py_version=''``
+- ``framework_version=''``
+
+The following example uses the ``image_name='763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.2-cpu-py37-ubuntu18.04'`` arg to specify the container image, Python version, and framework version:
+
+.. code:: python
+
+   tf_estimator = TensorFlow(entry_point='tf-train.py',
+                             role='SageMakerRole',
+                             train_instance_count=1,
+                             train_instance_type='ml.p2.xlarge',
+                             image_name='763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.2-cpu-py37-ubuntu18.04',
+                            script_mode=True)
+
```

Comment:
it might be worth adding a note here that if the user then uses the estimator for inference, they need to specify a new image URI.

also, should there be a corresponding section for inference?

Reply:
Removed training image to make it more general